Formula/sord.rb

```diff
@@ -20,13 +20,14 @@ class Sord < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "python@3.10" => :build
   depends_on "pcre"
   depends_on "serd"
 
   def install
-    system "./waf", "configure", "--prefix=#{prefix}"
-    system "./waf"
-    system "./waf", "install"
+    system Formula["python@3.10"].opt_bin/"python3", "./waf", "configure", "--prefix=#{prefix}"
+    system Formula["python@3.10"].opt_bin/"python3", "./waf"
+    system Formula["python@3.10"].opt_bin/"python3", "./waf", "install"
```

Comment:
Do we need to be this defensive about the version of Python3 used?

Reply:
Keg-only formulae are in `PATH` during the build, so just calling `python3` will call `python3.10`. The only time there's uncertainty about that is if one of the runtime dependencies have a Python3 dependency that isn't `python@3.10`, but even then those should be appended to `PATH`.